c7n/filters/core.py

```diff
@@ -1018,3 +1020,123 @@ def reorder(self, items, key=None):
             return items[::-1]
         else:
             return sorted(items, key=key, reverse=(self.order == 'desc'))
+
+
+class FakeModel:
+    id = 'c7n:_id'
+
+
+class FakeFilterRegistry(FilterRegistry):
```

Comment:
rather than fake, can we rename these item, fakes imply some support of testing infra.